src/main/resources/samples/JiraProjectKey.groovy

```diff
@@ -0,0 +1,10 @@
+@Grab(group='org.codehaus.groovy.modules.http-builder', module='http-builder', version='0.7' )
```

Comment:
It seems that the first line is not needed here.

Reply:
this line is added by the build